forloop_modules/function_handlers/auxilliary/abstract_function_handler.py

```diff
@@ -364,17 +364,17 @@ def export_internal_function(self, function_name):
 
         return code
     
-    def generate_shown_dataframe_option_field(self, df_variable_name, fields=None):
-        if fields is None:
-            fields = []
-
+    def generate_shown_dataframe_option_field(self, df_variable_name: str, fields: list[dict]):
```

Comment:
Are you sure fields always exist in all cases? I would keep there the None condtion